.claude/commands/pr-comments.md

```
@@ -0,0 +1,69 @@
+---
+description: "Show PR review comments"
+argument-hint: "<pr-number>"
+---
+
+Fetch and summarize all review comments for PR #$ARGUMENTS.
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)